tox.ini

```
@@ -19,7 +19,7 @@ deps =
 extras =
   mpl
   graphviz
-passenv = RETWORKX_TEST_PRESERVE_IMAGES RUSTWORKX_PKG_NAME
+passenv = RETWORKX_TEST_PRESERVE_IMAGES, RUSTWORKX_PKG_NAME
```

Instead of the comma syntax I tend to prefer the newline approach since it's more broadly compatible with more versions of tox:

```suggestion
passenv =
  RETWORKX_TEST_PRESERVE_IMAGES
  RUSTWORKX_PKG_NAME
```